### PR TITLE
dockerless build testing

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -209,4 +219,4 @@ import:
 # yours that I suspect aren't pinning because vendor.conf is respected
 # yours: containers/image.  No idea why this isn't self-pinning.
 - package: github.com/containers/storage
-  version: 47536c89fcc545a87745e1a1573addc439409165
+  version: master
```
Comment (deads2k) [inline glide.yaml]: We don't do this.  We do not depend on a master branch. You may use a semver branch, a tag, or a specific sha.  But not master.

File: glide.yaml [hunk 2]
```diff
@@ -191,6 +195,12 @@ import:
 # etcd pins a very old version that has contention issues
 - package: github.com/google/btree
   version: master
+# builds
+- package: github.com/projectatomic/buildah
+  repo: https://github.com/nalind/buildah-1.git
+  version: 979+980
+- package: github.com/seccomp/libseccomp-golang
+  version: master
```
Comment (deads2k) [inline glide.yaml]: We don't do this.  We do not depend on a master branch. You may use a semver branch, a tag, or a specific sha.  But not master.

**Comment by deads2k** [comment]
The new dependencies stand out. Are these used by the builder images themselves or the build controllers?  If it's just the builder images, this introduces more risk of transitive conflicts, untested transitive combinations, and eventual gridlock with other required updates.

We're already creating dozens of new images in new repos.  Rather than add dependencies to all of origin, it seems like it makes more sense to move the separate images with different requirements out of tree.

File: cmd/oc/oc.go [hunk 3]
```diff
@@ -20,6 +21,10 @@ import (
 )
 
 func main() {
+	if reexec.Init() {
```
Comment (deads2k) [inline cmd/oc/oc.go]: In `oc`?  This doesn't look right.  If you must have it for a specific command, localize the change there.

**Comment by smarterclayton** [comment]
Yeah, I think I'd prefer to split the builder image with this change into its own repo.  It's time, assuming you've got the right cuts to the builder core stuff.

**Pull request closed without merging**